mlserver/context.py

```diff
@@ -3,21 +3,21 @@
 
 from .settings import ModelSettings
 
-model_name_var: ContextVar[str] = ContextVar("model_name")
-model_version_var: ContextVar[str] = ContextVar("model_version")
+MODEL_NAME_VAR: ContextVar[str] = ContextVar("model_name")
+MODEL_VERSION_VAR: ContextVar[str] = ContextVar("model_version")
```

Comment:
nit: these are not constants and just global objects, so I'd guess PEP8 would recommend keeping them as lowercase? This is super nitpicky though, so you can just ignore it.